TASVideos/wwwroot/css/partials/_customizations.scss

```diff
@@ -83,6 +83,29 @@ article.wiki table {
 	}
 }
 
+/* Forum Tables */
+.postbody table {
+	width: 100%;
+	margin-bottom: 1rem;
+	color: var(--bs-dark);
+	border-collapse: collapse;
+}
+
+.postbody th, td {
+		padding: 0.3rem;
+		vertical-align: top;
+		border: 1px solid var(--bs-gray-600);
+}
+
+.postbody tbody th {
+		background-color: rgba(var(--bs-black-rgb), 0.05);
+}
+
+.postbody tbody + tbody {
+		border-top: 2px solid #dee2e6;
```

Comment:
I object to the use of a random inline style color.  Is there not a color that is "close enough" that is already defined? and if not, we need to put this into the color variables.

We want to consider color-blindness and other accessibility concerns.  Step 1 is to minimize the colors used, and Step 2 is to have them all defined in one place, so they can easily be tweaked as needed